server/src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -1073,9 +1070,9 @@ private boolean assertDocDoesNotExist(final Index index, final boolean allowDele
     private void updateDocs(final Term uid, final List<ParseContext.Document> docs, final IndexWriter indexWriter) throws IOException {
         if (softDeleteEnabled) {
             if (docs.size() > 1) {
-                indexWriter.softUpdateDocuments(uid, docs, Lucene.getSoftDeleteDVMarker());
+                indexWriter.softUpdateDocuments(uid, docs, Lucene.newSoftDeleteField());
```

Comment:
I think we can cache this in here in a constant so we don't need to re-create a new one every time.